pages/form-builder/support/[[...supportType]].tsx

```diff
@@ -33,400 +34,457 @@ export default function Contactus() {
+                      id: "request-form",
+                      name: "form",
+                      label: t("support.request.option3"),
+                      required: true,
+                    },
+                    {
+                      id: "request-other",
+                      name: "other",
+                      label: t("support.request.option4"),
+                      required: true,
+                    },
+                  ]}
+                ></MultipleChoiceGroup>
+              </fieldset>
+              <div className="focus-group">
+                <Label
+                  id={"label-description"}
+                  htmlFor={"description"}
+                  className="required"
+                  required
                 >
-                  <ol className="gc-ordered-list">
-                    {Object.entries(errors).map(([fieldKey, fieldValue]) => {
-                      return (
-                        <ErrorListItem
-                          key={`error-${fieldKey}`}
-                          errorKey={fieldKey}
-                          value={fieldValue}
-                        />
-                      );
-                    })}
-                  </ol>
-                </Alert>
-              )}
-              <form id="contactus" method="POST" onSubmit={handleSubmit} noValidate>
-                {content}
+                  {t("support.description.title")}
+                </Label>
+                <Description id={"description-description"}>
+                  {t("support.description.description")}
+                </Description>
+                <TextArea
+                  id={"description"}
+                  name={"description"}
+                  className="required w-[34rem] mt-4"
+                  required
+                  characterCountMessages={{
+                    part1: t("formElements.characterCount.part1"),
+                    part2: t("formElements.characterCount.part2"),
+                    part1Error: t("formElements.characterCount.part1-error"),
+                    part2Error: t("formElements.characterCount.part2-error"),
+                  }}
+                />
+              </div>
+
+              <Button type="submit" className="gc-button--blue" disabled={isSubmitting}>
+                {t("submitButton", { ns: "common" })}
+              </Button>
+            </form>
+          </>
+        )}
+      </Formik>
+    </>
+  );
+
+  return (
+    <>
+      {!isSuccessScreen && (supportType === "contactus" ? contactUsForm : supportForm)}
 
-                <Button type="submit" className="gc-button--blue" disabled={isSubmitting}>
-                  {t("submitButton", { ns: "common" })}
-                </Button>
-              </form>
-            </>
-          )}
-        </Formik>
-      )}
       {isSuccessScreen && (
         <>
-          <h1>{t("requestSuccess.title")}</h1>
+          <h1 ref={headingSuccessRef}>{t("requestSuccess.title")}</h1>
```

Comment:
Focusing the heading vs a live region around the "page" is a bit cleaner to let an assistive technology user know the page content has changed.